### PR TITLE
Lf 2669-after retiring a sensor it isnt immediately removed from the farm map

### PR DESCRIPTION
**Pull request by AbdusShaikh**
To test:
1. Upload a sensor if you don't already have on
2. Click on it in the map and retire it

You should see a success message snackbar and the sensor should no longer appear on the map

### CODE REVIEW

File: packages/api/src/controllers/sensorController.js [hunk 1]
```diff
@@ -565,7 +565,7 @@ const sensorController = {
   async retireSensor(req, res) {
     const trx = await transaction.start(Model.knex());
     try {
-      const { external_id, location_id, farm_id, partner_id, brand_name } = req.body.sensorInfo;
+      const { external_id, location_id, farm_id, partner_id, brand_name } = req.body;
```
Comment (fsayyid) [inline packages/api/src/controllers/sensorController.js:568]: Client should have to send this data in the request body. Just the `location_id` should be enough to unclaim the sensor.

Comment (fsayyid) [reply packages/api/src/controllers/sensorController.js:568]: You can pull the other data from the database.

**Review by fsayyid — APPROVED** [review]
This seems to be working in the non-esci sensor case. And I have a PR to fix the retiring an ESCI sensor bug.

**Comment by fsayyid** [comment]
Please deal with the merge conflicts

**Review by Cali0707 — CHANGES_REQUESTED** [review]
@AbdusShaikh I'm getting an error: `Uncaught ReferenceError: DEPTH_UNIT is not defined`

**Review by Cali0707 — CHANGES_REQUESTED** [review]
@AbdusShaikh when I try and retire an ESCI sensor nothing happens - seems to be working for non ESCI sensors though

**Comment by AbdusShaikh** [comment]
> @AbdusShaikh when I try and retire an ESCI sensor nothing happens - seems to be working for non ESCI sensors though

This issue was addressed by a seperate PR: [#2198](https://github.com/LiteFarmOrg/LiteFarm/pull/2198)

**Comment by fsayyid** [comment]
@AbdusShaikh need to merge integration into your branch

**Review by Cali0707 — APPROVED** [review]
This is working for me!

**Pull request merged**